hack/local-up-kubeedge.sh

```diff
@@ -40,6 +50,23 @@ function kind_up_cluster {
   kind create cluster ${CLUSTER_CONTEXT}
 }
 
+# install CNI plugins
+function install_cni_plugins {
+  if [[ "${CONTAINER_RUNTIME}" = "docker" ]]; then
+    # if we will use docker as edgecore container runtime, we do not need to install CNI plugins
+    # so just return
+    return
+  fi
+
+  # install CNI plugins if not exist
```

Comment:
Why do you need to install the cni plugin？ If user use containerd, he need install cni on every edge node?

Reply:
yes, https://github.com/containerd/containerd/blob/main/docs/getting-started.md#option-1-from-the-official-binaries

Reply:
If we do not install CNI, does it report err? since we do not use cni in edge node for now

Reply:
containerd cannot run or start pod on edge nodes
```
failed to setup network for sandbox "xxxxxxxxxxxxxx": failed to find plygin "loopback" in path [/opt/cni/bin]
```

Reply:
Got it, thanks for your explanation